docs/design_decisions.md

```diff
@@ -52,9 +52,18 @@ Voor de namen van endpoints, url's en parameters worden alleen kleine letters ge
   * kinderen (resource "ingeschrevenpersonen" en relatie "heeft kinderen")
 
 ### DD1.8 Namen van Identificatie properties zijn afhankelijk van het wel of niet voorkomen van sibling properties
-Wanneer een relatie-property (niet link of embedded) alleen de identificatie van een gerelateerde resource bevat en geen andere properties, wordt als naam van de property de naam van de resource plus het woord 'Identificatie' gebruikt.
+Wanneer een relatie-property (niet link of embedded) geen andere properties heeft dan alleen de identificatie van een gerelateerde resource, wordt deze niet gemodelleerd als property met daarbinnen de identificatie-property. De relatie-property wordt dan vervangen door de identificatie-property met als naam de naam van de gerelateerde resource plus het woord 'Identificatie'.
```

Comment:
Hierbij de overwegingen meenemen:
- voordeel ...Identificaties: eenvoudige structuur, eenvoudig te gebruiken
- voordeel object met identificatie: uitbreidbaar met andere properties zonder breaking change (zie in BRK wijziging van bijbehorendeAppartementrechtIdentificaties naar bijbehorendeAppartementrechten.identificatie t.b.v. het toevoegen van volgnummer en indicatieSluimerend)

Reply:
Ik heb de tekst zo aangepast dat deze tot afweging uitnodigt.